src/constants/currency.js

```diff
@@ -40,6 +76,41 @@ export const CRYPTO_CURRENCY_MAP = keyBy(CRYPTO_CURRENCY);
 export const FIAT_CURRENCY = ['USD', 'AUD'];
 export const FIAT_CURRENCY_MAP = keyBy(FIAT_CURRENCY);
 
-export const isSynth = currency => !!SYNTHS_MAP[currency];
-export const isCryptoCurrency = currency => !!CRYPTO_CURRENCY_MAP[currency];
-export const isFiatCurrency = currency => !!FIAT_CURRENCY_MAP[currency];
+export const currencyKeyToIconMap = {
+	[CRYPTO_CURRENCY_MAP.ETH]: ETHIcon,
```

Comment:
I think we really need to find a better solution for that soon. I'm scared we'll forget adding new synths..etc. Or maybe write tests which compare the list of synths coming from synthetix-js and make sure we've got the same number.

Reply:
so I can break on CircleCi in case we forgot to add an extra line